build/webxr-polyfill.module.js

```diff
@@ -141,19 +141,68 @@ class EventTarget {
 }
```

Comment:
We can save checking in the built files until we're ready to merge back to master